website/blog/2023-03-03-databend-weekly-83.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "This Week in Databend #83"
+date: 2023-03-03
+slug: 2023-03-03-databend-weekly
+tags: [databend, weekly]
+contributors:
+- name: andylokandy
```

Comment:
Most data here are duplicated. Maybe we can generate them by only input username.

Reply:
Let's improve it later

Reply:
A simple improvement, with a special judgment on `mergify[bot]` .